src/components/ContentBlocks/BlockVideoTitleTextButton/BlockVideoTitleTextButton.tsx

```diff
@@ -31,22 +31,21 @@ export const BlockVideoTitleTextButton: FunctionComponent<BlockVideoTitleTextBut
 						{/* 16 by 9 => 100% by 56% */}
 						<div className="c-video-wrapper" style={{ paddingBottom: '56%' }}>
 							{/* TODO replace this with the flowplayer video component */}
-							<video src={videoSource} />
+							<video src={videoSource} controls={true} />
```

Comment:
`controls` could replace `controls={true}`.

Reply:
Video gaat doorgeefbaar moeten zijn via slots/children, aangezien dat flowplayer moet kunnen zijn?

Reply:
flowplayer zal een component zijn in de components aangezien we die ook moeten kunnen tonen in de admin app
dus dan kan die hier rechtstreeks getoont worden zonder slot
het token systeem zal wel in de client zitten